arm-resources/appliances/2016-09-01-preview/swagger/appliances.json

```diff
@@ -0,0 +1,1262 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ManagedApplicationClient",
+    "version": "2016-09-01-preview",
+    "description": "ARM managed applications (appliances)"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
Please add `x-ms-examples` to all the APIs: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md.